frontend/src/store/eam/reducer.ts

```diff
@@ -4,20 +4,23 @@ import { getWorkers, loadGroups } from './actions';
 import {
   EAMGroupGetByTenantResponseItemDto,
   EAMWorkerGetAllItemResponseDto,
+  EAMTenantByIdResponseDto,
 } from 'common/types/types';
 
 type State = {
   dataStatus: DataStatus;
   workers: EAMWorkerGetAllItemResponseDto[];
   groupsDataStatus: DataStatus;
   groups: EAMGroupGetByTenantResponseItemDto[];
+  tenant: EAMTenantByIdResponseDto | null;
```

Comment:
```suggestion
```
we don't want to keep a tenant here. we should keep a tenant in the `app` reducer, but action should for changing should be in the `eam` scoop